### ISSUE TITLE
Add a simple "tagging" feature to activities

### ISSUE THREAD

**Issue by Unicorn-rzl** [jonasoreland/runnerup#213]
tl;dr: Add customizable "tags" below the notes field, that enter text into said field.

Longer:
Had some idea for a feature today, don't know whether this is interesting for others. Feel free to ignore it, if you don't think anybody else needs it.

The idea would be to add some basic "tagging" interface below the "note" field of an activity. You could add tags like "rainy", "cold", "With Jonas", the used shoe or whatever else in the configuration and they show up below the notes field. When you click on them, the tag gets entered into the notes field (e.g. old_notes + "#rainy" + "#Adidazunosics_MegaBrook_9001").

Bonus: Make those tags also color customizable so that you can "visually sort" them if you want/need to.

Background: I often run together with friends and usually add their names as a note to an activity. Would be easier if I just click their names after a run...
